<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesDSS</class>
 <widget class="QMainWindow" name="NotesDSS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotesDSS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_pages">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Документы</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Акт оприходования</string>
     </property>
     <addaction name="action_13"/>
     <addaction name="action_14"/>
     <addaction name="action_15"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Акт списания</string>
     </property>
     <addaction name="action_17"/>
     <addaction name="action_20"/>
     <addaction name="action_18"/>
     <addaction name="action_21"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
    <addaction name="action_22"/>
    <addaction name="action_23"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="aFirms"/>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="aUnits"/>
    <addaction name="separator"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Отчеты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Материалы</string>
   </property>
  </action>
  <action name="aUnits">
   <property name="text">
    <string>Единицы измерения</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Статистика</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Объекты</string>
   </property>
  </action>
  <action name="aFirms">
   <property name="text">
    <string>Контрагенты</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Покупка</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Пришло в ремонт</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Вернулось после ремонта, проверки</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Продажа</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Отправка в ремонт</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Отправка после ремонта</string>
   </property>
  </action>
  <action name="action_21">
   <property name="text">
    <string>Брак</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>Требование</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>Производство</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
